seqseek/chromosome.py

```diff
@@ -45,14 +46,18 @@ def validate_name(self):
         if self.name not in self.chromosome_lengths.keys():
             raise ValueError("{name} is not a valid chromosome name".format(name=self.name))
 
-    def validate_coordinates(self, start, end):
-        if start < 0 or end < 0:
-            raise ValueError("Start and end must be positive integers")
+    def validate_coordinates(self, start, end, loop=False):
```

Comment:
"loop" feels much more like a property of Chromosome than a function parameter. I struggle to think of when you'd want to modify loop on a per-function-call basis rather than when you're constructing the MT chromosome.